addons/web/models/models.py

```diff
@@ -1052,6 +1052,43 @@ def web_override_translations(self, values):
                 translations[self.env.lang or 'en_US'] = values[field_name]
                 self.update_field_translations(field_name, translations)
 
+    def _read_property(self, field_name, property_name=None):
+        """Read the given properties, and execute all checks.
+
+        The goal of that method is to have a similar API than the getter
+        on the standard fields (that can then be used in template, etc).
+        """
+        if self:
+            self.ensure_one()
+
+        if not property_name:
+            return self.read([field_name])[0][field_name] if self else False
```

Comment:
This method should be only called for properties field. Don't create a new way to get a scalar value in a very inefficient way.